massa-network/src/settings.rs

```diff
@@ -90,6 +95,21 @@ mod tests {
                 initial_peers_file: std::path::PathBuf::new(),
                 peer_list_send_timeout: MassaTime::from(500),
                 max_in_connection_overflow: 2,
+                bootstrap_peers_config: PeerTypeConnectionConfig {
+                    target_out: 1,
+                    max_out_attempts: 1,
+                    max_in: 1,
+                },
+                whitelist_peers_config: PeerTypeConnectionConfig {
```

Comment:
```
whitelist_peers_config:
  max_in: 10,
  target_out: 10,
  max_out_attempts: 10,
```

Let's be more explicit:
* max_in => max_in_connections
* target_out => target _out_connections

Also maybe it's worth specifying the max number of open nodes per IP address for each type to give more granularity.

Reply:
> Also maybe it's worth specifying the max number of open nodes per IP address for each type to give more granularity.

You mean the max_in_connections_per_ip in Network settings would become PeerType specific ?

Reply:
> ```rust
> whitelist_peers_config:
>   max_in: 10,
>   target_out: 10,
>   max_out_attempts: 10,
> ```

it's already the current config values in this file, I update it in the config.toml

Reply:
> > Also maybe it's worth specifying the max number of open nodes per IP address for each type to give more granularity.
> 
> You mean the max_in_connections_per_ip in Network settings would become PeerType specific ?

maybe as a followup

Reply:
> > ```rust
> > whitelist_peers_config:
> >   max_in: 10,
> >   target_out: 10,
> >   max_out_attempts: 10,
> > ```
> 
> it's already the current config values in this file, I update it in the config.toml

? I didn't understand this comment.

What I meant is just to rename the fields to make them more explicit (max_in_connections instead of max_in)

Reply:
> What I meant is just to rename the fields to make them more explicit (max_in_connections instead of max_in)

My concern was about the config values in themselves. Are they the expected ones ? (especially for the whitelist ones as there was no reference value)

Reply:
> > > 
> > 
> > 
> > You mean the max_in_connections_per_ip in Network settings would become PeerType specific ?
> 
> maybe as a followup

See #2333

Reply:
I would go for max_in: 3, target_out: 2, max_out_attempts: 2